pandas/core/indexing.py

```diff
@@ -656,13 +666,12 @@ def _align_series(self, indexer, ser: ABCSeries, multiindex_indexer: bool = Fals
         Parameters
         ----------
         indexer : tuple, slice, scalar
-            The indexer used to get the locations that will be set to
-            `ser`
+            Indexer used to get the locations that will be set to `ser`.
         ser : pd.Series
-            The values to assign to the locations specified by `indexer`
-        multiindex_indexer : boolean, optional
-            Defaults to False. Should be set to True if `indexer` was from
-            a `pd.MultiIndex`, to avoid unnecessary broadcasting.
+            Values to assign to the locations specified by `indexer`.
+        multiindex_indexer : bool, default False, optional
+            `multiindex_indexer` should be set to True if `indexer`, was
```

Comment:
I think this would fail validation for not starting with a capital letter; probably best to revert this